lectures/symbolic_sympy.md

```diff
@@ -0,0 +1,272 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.5
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Introduction to Symbolic Algebra and Calculus with Sympy
+
+
+## Overview
+
+Symbolic computation, or computer algebra, involves using algorithms and software to perform exact computations and manipulate mathematical equations. While **Mathematica** is a popular tool for symbolic computations, it's proprietary and can be costly. As an open-source alternative, the **sympy** library in Python offers a comprehensive set of functionalities for symbolic mathematics. This allows users to perform a range of operations, from algebraic manipulations to calculus, providing exact solutions instead of numerical approximations.
```

Comment:
Each sentence should be one paragraph. Also, when possible, please add a link to the documentation of the library that is mentioned (i.e., [sympy](https://www.sympy.org/en/index.html))